monitoring/exporters/elasticsearch/docker-entrypoint.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/bin/elasticsearch_exporter --es.uri="${audius_elasticsearch_url}"
```

Comment:
Do we need custom docker image here, can't we just override command since its simple?

Reply:
`bash -c ""`

Reply:
These images only have `sh` and that syntax didn't work for me.

Reply:
did you try sh -c?

Reply:
that syntax didn't work for me.

Reply:
I tested this just now and it works,

```
docker run -it --entrypoint "/bin/sh" quay.io/prometheuscommunity/elasticsearch-exporter:v1.3.0 -c "echo 1"
```

overriding entrypoint in docker compose: https://docs.docker.com/compose/compose-file/#entrypoint

Reply:
so something like `entrypoint: ["/bin/sh", "-c", "<cmd>"]`

Reply:
Your test isn't representative of the use case.

Your test should have been similar to these changes:

```
diff --git a/common-services.yml b/common-services.yml
index de54826..8797b0f 100644
--- a/common-services.yml
+++ b/common-services.yml
@@ -53,7 +53,7 @@ services:
     restart: always

   exporter_elasticsearch:
-    image: audius/exporter-elasticsearch:v1.3.0
+    image: quay.io/prometheuscommunity/elasticsearch-exporter:v1.3.0
     restart: always

   exporter_redis:
diff --git a/discovery-provider/docker-compose.yml b/discovery-provider/docker-compose.yml
index ef15eb9..9e9f32f 100644
--- a/discovery-provider/docker-compose.yml
+++ b/discovery-provider/docker-compose.yml
@@ -51,6 +51,7 @@ services:
     env_file:
       - ${NETWORK:-prod}.env
       - override.env
+    entrypoint: ["/bin/sh", "-c", "/bin/elasticsearch_exporter", '--es.uri="${audius_elasticsearch_url}"']
     networks:
       - discovery-provider-network

```

Which results in these logs:

```
$ d logs cad5dc120489 | head
level=info ts=2022-10-05T17:05:45.276090035Z caller=clusterinfo.go:214 msg="triggering initial cluster info call"
level=info ts=2022-10-05T17:05:45.276295045Z caller=clusterinfo.go:183 msg="providing consumers with updated cluster info label"
level=error ts=2022-10-05T17:05:45.277112333Z caller=clusterinfo.go:267 msg="failed to get cluster info" err="Get \"http://localhost:9200/\": dial tcp 127.0.0.1:9200: connect: connection refused"
```

Reply:
prob should resolve this on huddle, long gh threads slow things down

Reply:
@cheran-senthil is there anything else left to discuss here? if not, we can resolve this thread.

Reply:
You did

> ```
> entrypoint: ["/bin/sh", "-c", "/bin/elasticsearch_exporter", '--es.uri="${audius_elasticsearch_url}"']
> ```

Solution
`entrypoint: ["/bin/sh", "-c", '/bin/elasticsearch_exporter --es.uri="$$audius_elasticsearch_url"'] `